crates/bevy_render/src/texture/ktx2.rs

```diff
@@ -1363,100 +1242,184 @@ pub fn ktx2_format_to_texture_format(
         ktx2::Format::EAC_R11G11_SNORM_BLOCK => TextureFormat::EacRg11Snorm,
         ktx2::Format::ASTC_4x4_UNORM_BLOCK | ktx2::Format::ASTC_4x4_SRGB_BLOCK => {
             if is_srgb {
-                TextureFormat::Astc4x4RgbaUnormSrgb
+                TextureFormat::Astc {
+                    block: AstcBlock::B4x4,
+                    channel: AstcChannel::UnormSrgb,
+                }
             } else {
-                TextureFormat::Astc4x4RgbaUnorm
+                TextureFormat::Astc {
+                    block: AstcBlock::B4x4,
+                    channel: AstcChannel::Unorm,
+                }
```

Comment:
Maybe collapse these in the same way as above so only the `channel` member has its value assigned based on the `if is_srgb` as the rest is the same.

Reply:
And for all the ones below.